language_dump.py

```diff
@@ -63,10 +69,18 @@ def dump_translations():
     fallback_language = args.fallback
     verbose = args.verbose
     strings_by_object = {}
+    reference_str_count = 0
+    translated_str_count = 0
 
     for filename in glob.iglob(args.objects + '/**/*.json', recursive=True):
-        extract_translations(language_to_extract, fallback_language, verbose,\
-                             filename, strings_by_object)
+        obj_translations, ref_obj_translation =\
+            extract_translations(language_to_extract, fallback_language, verbose,\
+                                 filename, strings_by_object)
+        reference_str_count += ref_obj_translation
+        translated_str_count += obj_translations
+
+    print('{}: {}% completeness'.format(language_to_extract,
+                                        round(100*translated_str_count/reference_str_count, 2)))
```

Comment:
The formatting here is a little awkward. My suggestion would be to combine local assignments with f-strings again.

Reply:
Right, I forgot to update when rebasing, done now.